infra/modules/azure_storage_account/alerts.tf

```diff
@@ -0,0 +1,35 @@
+# -----------------------------------------------
+# Alerts
+# -----------------------------------------------
+
+resource "azurerm_monitor_metric_alert" "storage_account_health_check" {
+  count               = local.tier_features.advanced_threat_protection ? 1 : 0
+  name                = "[${replace("${var.environment.domain != null ? "${var.environment.domain} | " : ""}${azurerm_storage_account.this.name}", "-", "")}] Low Availability"
```

Comment:
Should we use the domain name here? I'm sticking to use the resource name only - just to know

Reply:
I wasn't sure actually, we can surely stick to the resource name that already contains all the information